kotlinx-collections-immutable/src/main/kotlin/kotlinx/collections/immutable/implementations/immutableList/PersistentVectorBuilder.kt

```diff
@@ -26,12 +26,16 @@ private class Marker // TODO: Rename to MutabilityOwnership?
 class PersistentVectorBuilder<E>(private var vector: PersistentList<E>,
                                  private var vectorRoot: Array<Any?>?,
                                  private var vectorTail: Array<Any?>,
-                                 private var rootShift: Int) : AbstractMutableList<E>(), PersistentList.Builder<E> {
+                                 internal var rootShift: Int) : AbstractMutableList<E>(), PersistentList.Builder<E> {
     private var marker = Marker()
-    private var root = vectorRoot
-    private var tail = vectorTail
+    internal var root = vectorRoot
+        private set
+    internal var tail = vectorTail
+        private set
     override var size = vector.size
```

Comment:
This doesn't seem to be changed from outside too

Reply:
+1